tests/fixtures/gengo/job_details_ready.html

```
@@ -0,0 +1,5 @@
+<html>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)